pandas/core/computation/eval.py

```diff
@@ -220,6 +220,10 @@ def eval(expr, parser='pandas', engine=None, truediv=True,
         If `target` is provided, and the expression mutates `target`, whether
         to modify `target` inplace. Otherwise, return a copy of `target` with
         the mutation.
+    partial_str_match : bool, optional, default False
+        If this is True, an `expr` like "string_query in list_like_of_strings"
+        is interpreted as partial string match (the default behavior is exact
+        matching).
```

Comment:
Add a "versionadded" tag in the docstring.

Reply:
Thank you, I added.